scripts/dev/ci/travis_install.sh

```diff
@@ -111,18 +112,18 @@ tox --version
 case $TESTENV in
     py34-cov)
         pip_install codecov
-        apt_install xvfb $pyqt_pkgs libpython3.4-dev
+        apt_install xvfb "$pyqt_pkgs" libpython3.4-dev
         check_pyqt
         ;;
     pylint|vulture)
-        apt_install $pyqt_pkgs libpython3.4-dev
+        apt_install "$pyqt_pkgs" libpython3.4-dev
         check_pyqt
         ;;
     flake8)
         apt_install libpython3.4-dev
         ;;
     docs)
-        apt_install $pyqt_pkgs asciidoc libpython3.4-dev
+        apt_install "$pyqt_pkgs" asciidoc libpython3.4-dev
```

Comment:
Those all are intentionally unquoted. Though I guess `pyqt_pkgs` could be an array and `"${pyqt_pkgs[@]}"` used here.